flink-formats/flink-avro/src/test/java/org/apache/flink/table/runtime/batch/AvroTypesITCase.java

```diff
@@ -147,24 +150,26 @@ public AvroTypesITCase(
 
 	@Test
 	public void testAvroToRow() throws Exception {
-		ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
+		StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
 		env.getConfig().registerTypeWithKryoSerializer(LocalDate.class, AvroKryoSerializerUtils.JodaLocalDateSerializer.class);
```

Comment:
These should not be necessary in our tests, right?

Reply:
We do need those in a0122e0 as we still use the joda time in this commit. We can remove them, (and we do in the next commit).

The default Kryo serialization does not work well.

Reply:
BTW, after the comment, I removed the `JodaLocalDateSerializer` and `JodaLocalTimeSerializer` classes in 453d02c as they are no longer used.